blockchain/src/envVarsSchema.js

```diff
@@ -0,0 +1,156 @@
+const Joi = require("joi");
+
+const envVarsSchema = Joi.object({
+  PORT: Joi.number().port().default(8085).note("This is the port where the multichain can be downloaded (backup)"),
+  ORGANIZATION: Joi.string()
+    .default("MyOrga")
+    .note(
+      "In the blockchain network, each node is represented by its organization name. This environment variable sets this organization name. It is used to create the organization stream on the blockchain.",
+    ),
+  MULTICHAIN_RPC_PORT: Joi.number()
+    .port()
+    .default(8000)
+    .required()
+    .note(
+      "The port used to expose the multichain daemon of your Trubudget blockchain installation(bc). The port used to connect to the multichain daemon(api). This will be used internally for the communication between the API and the multichain daemon.",
+    ),
+  MULTICHAIN_RPC_USER: Joi.string()
```

Comment:
Pipeline fails because MULTICHAIN_RPC_USER is probably not set explicitly, and doesn't default. Maybe .allow("") ?
Error: Config validation error: "MULTICHAIN_RPC_USER" is required
    at Object.<anonymous> (/home/node/src/config.js:5:9)
    at Module._compile (node:internal/modules/cjs/loader:1364:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)
    at Module.load (node:internal/modules/cjs/loader:1203:32)
    at Module._load (node:internal/modules/cjs/loader:1019:12)
    at Module.require (node:internal/modules/cjs/loader:1231:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/home/node/src/index.js:25:16)
    at Module._compile (node:internal/modules/cjs/loader:1364:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)